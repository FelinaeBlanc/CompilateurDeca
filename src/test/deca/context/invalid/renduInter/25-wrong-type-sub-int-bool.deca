{
    int x = 4;
    boolean y = true;
    x = x-y;
}