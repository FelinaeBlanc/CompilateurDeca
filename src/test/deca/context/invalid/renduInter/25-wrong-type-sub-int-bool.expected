25-wrong-type-sub-int-bool.deca:4:8: Error unsupported types